doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-test-template-rule-based-proc.adoc

```diff
@@ -10,22 +10,27 @@ Create a test scenario template for rule-based test scenarios by following the p
 . Select *RULE* as the *Source type*.
 . Click *Ok* to create and open the test scenario in the test scenarios designer.
 . To map the *GIVEN* column header to a data object,
-.. Click an instance header in the *GIVEN* section.
+.. Click an instance header cell in the *GIVEN* section.
 .. Select the data object from the *Test Tools* tab.
-.. Click *Add*.
-. To insert more properties of the data object, right-click the property header and select *Insert column right* or *Insert column left* as required.
-. To map a data object field to a property cell,
-.. Click a property cell.
-.. Select the data object field from the *Test Tools* tab.
-.. Click *Add*.
+.. Click *Insert Data Object*.
+.. Optional: Click *Clear selection* to remove the selected data object.
 . To map the *EXPECT* column header to a data object,
-.. Click an instance header in the *EXPECT* section.
+.. Click an instance header cell in the *EXPECT* section.
 .. Select the data object from the *Test Tools* tab.
-.. Click *Add*.
-. To insert more properties of the data object, right-click the property header and select *Insert column right* or *Insert column left* as required.
+.. Click *Insert Data Object*.
+.. Optional: Click *Clear selection* to remove the selected data object.
 . To map a data object field to a property cell,
-.. Click a property cell.
+.. Click an instance header cell or property header cell.
```

Comment:
Suggestion - Replace "Click" with "Select" for non-button UI elements:

s/Click an instance header cell/Select an instance header cell